R/splatPop-simulate.R

```diff
@@ -371,15 +360,25 @@ splatPopSimulateSC <- function(sim.means,
 #' @importFrom SingleCellExperiment SingleCellExperiment
 splatPopSimulateSample <- function(params = newSplatPopParams(),
                                    method = c("single", "groups", "paths"),
+                                   batch = "batch1",
                                    counts.only = FALSE,
                                    verbose = TRUE,
                                    sample.means, ...) {
 
     method <- match.arg(method)
-    set.seed(getParam(params, "seed"))
 
     # Get the parameters we are going to use
-    nCells <- getParam(params, "nCells")
+    if(getParam(params, "nCells.sample")){
```

Comment:
Probably safer to test `is.null` or `is.data.frame` (or similar) rather than just using the object.

Reply:
The nCells.sample parameter is a logical TRUE/FALSE. Should I specify with "nCells.sample == TRUE"?

Reply:
Ah, ok! I assumed it was a vector of number of cells for each sample or something. This is fine then (or `isTRUE(...)` if you want to make it really clear).

Reply:
Updated to isTRUE